blocks/api/raw-handling/utils.js

```diff
@@ -99,7 +104,7 @@ function isInlineForTag( nodeName, tagName ) {
 
 export function isInline( node, tagName ) {
 	const nodeName = node.nodeName.toLowerCase();
-	return !! inlineWhitelist[ nodeName ] || isInlineForTag( nodeName, tagName );
+	return !! inlineWhitelist.hasOwnProperty( nodeName ) || isInlineForTag( nodeName, tagName );
```

Comment:
The type coercion is unnecessary (`hasOwnProperty` will certainly return a boolean).